modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -85,7 +110,7 @@ Example DataWeave 2.0 expression to be used when configuring the policy:
 #[attributes.queryParams.'client_secret']
 ----
 
```

Comment:
Note that in this case the requester needs to send the two specified query parameters. Although this is a supported configuration, it is not recommended to send the credentials as query parameters since this could pose a security risk. The recommended practice is to use headers.

Reply:
Done! Thanks!